butane_core/src/codegen/mod.rs

```diff
@@ -404,7 +404,7 @@ pub fn get_deferred_sql_type(ty: &syn::Type) -> DeferredSqlType {
         .or_else(|| get_autopk_sql_type(ty))
         .unwrap_or_else(|| {
             DeferredSqlType::Deferred(TypeKey::CustomType(
-                ty.clone().into_token_stream().to_string(),
+                ty.clone().into_token_stream().to_string().replace(' ', ""),
```

Comment:
another approach here would be to trim any `foo::bar::` namespace prefixes, so that there can only be a single `Baz` in any namespace.